Sources/Utils/POSIX/Stringify.cpp

```diff
@@ -17,8 +17,8 @@
 #include <cerrno>
 #include <csignal>
 // clang-format off
-#include <sys/types.h>
-#include <sys/ptrace.h>
+#include <sys/types.h> // NOLINT
```

Comment:
Why is this required?

Reply:
I don't know the fundamental reason why, but you added the `clang-format off` in 2016 because types needed to be included before ptrace.

Reply:
That worked.